ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: ProjectTemplate
+metadata:
+  name: default-project-template
+spec:
+  subjects:
+    - kind: User
+      name: admin@cluster
+      role: Admin
+    - kind: User
+      name: user@cluster
+      role: User
+  namespaceMetadata:
+    annotations:
+      extended-monitoring.deckhouse.io/enabled: ""
+    labels:
+      created-from-project-type: default-project-template
+  valuesSchema:
+    requests:
+      type: object
+      properties:
+        cpu:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: "^[0-9]+m?$"
+        memory:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
+        storage:
+          type: string
+          pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
+    limits:
+      type: object
+      properties:
+        cpu:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: "^[0-9]+m?$"
+        memory:
+          oneOf:
+            - type: number
+              format: int
+            - type: string
+          pattern: '^[0-9]+(\.[0-9]+)?(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)?$'
+  template: |
+    ---
+    # Max requests and limits for resource and storage consumption for all pods in a namespace.
+    # Refer to https://kubernetes.io/docs/concepts/policy/resource-quotas/
+    apiVersion: v1
+    kind: ResourceQuota
+    metadata:
+      name: all-pods
+    spec:
+      hard:
+        {{ with .templateValues.requests.cpu }}requests.cpu: {{ . }}{{ end }}
+        {{ with .templateValues.requests.memory }}requests.memory: {{ . }}{{ end }}
+        {{ with .templateValues.requests.storage }}requests.storage: {{ . }}{{ end }}
+        {{ with .templateValues.limits.cpu }}limits.cpu: {{ . }}{{ end }}
+        {{ with .templateValues.limits.memory }}limits.memory: {{ . }}{{ end }}
+    ---
+    # Max requests and limits for resource consumption per pod in a namespace.
+    # All the containers in a namespace must have requests and limits specified.
+    # Refer to https://kubernetes.io/docs/concepts/policy/limit-range/
+    apiVersion: v1
+    kind: LimitRange
+    metadata:
+      name: all-containers
+    spec:
+      limits:
+        - max:
+            {{ with .templateValues.limits.cpu }}cpu: {{ . }}{{ end }}
+            {{ with .templateValues.limits.memory }}memory: {{ . }}{{ end }}
+          maxLimitRequestRatio:
+            cpu: 1
+            memory: 1
+          type: Container
```

Comment:
Let's not create the limit range for now. The behavior of this resource is a real puzzle.
```suggestion
```

Instead, let's provide an example with the operation policy to enforce setting requests by a user.
https://deckhouse.io/documentation/v1/modules/015-admission-policy-engine/cr.html#operationpolicy-v1alpha1-spec-policies-requiredresources